user-group/announcement-01.html

```diff
@@ -0,0 +1,101 @@
+                                                            <p style="font-family:sans-serif;font-weight:normal;text-align:center;font-size:16px;line-height:22px;Margin-top:0;Margin-bottom:30px;color:#111419;"><span style="color:#111419">Ally will be represented at the 2017 CSUN conference in San Diego. Visit our booth in the exhibitor hall or attend our session "Unpacking Course Content Accessibility: Insights, Trends and Case Studies</span></p>
+                                                        </td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td style="text-align:center;padding-top:0;padding-bottom:0;padding-right:60px;padding-left:60px;vertical-align:top;height:80px">
+                                                            <a href="http://www.csun.edu/cod/conference/2017/sessions/index.php/public/presentations/view/294" target="_blank" style="padding-top:10px;padding-bottom:10px;padding-right:20px;padding-left:20px;background-color:#007900;font-family:sans-serif;font-weight:lighter;font-size:22px;line-height:22px;text-align:left;text-decoration: none;color:#FFFFFF;">View session details</a>
+                                                        </td>
+                                                    </tr>
+                                                </tbody>
+                                            </table>
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td bgcolor="#EBECEC" style="padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;Margin:0 auto;" width="600">
+                                            <table style="border-spacing:0;" width="100%">
+                                                <tbody>
+                                                    <tr>
+                                                        <td style="text-align:center;padding-top:30px;padding-bottom:0;padding-right:60px;padding-left:60px;vertical-align:top;">
+                                                            <h3 style="font-family:sans-serif;font-weight:bold;text-align:center;font-size:22px;line-height:28px;Margin-top:0;Margin-bottom:10px;color:#00c0cb;"><span style="color:#111419">User Group Survey In Progress</span></h2>
+                                                            <p style="font-family:sans-serif;font-weight:normal;text-align:center;font-size:16px;line-height:22px;Margin-top:0;Margin-bottom:30px;color:#111419;"><span style="color:#111419">We've found from 70 responses (so far), 80% say that events and training is the main source of info for accessibility of course content. Is that true for your institution?</span></p>
```

Comment:
2 suggestions:

1. This one is somewhat important: `80% say that events and training is the main source...` I think should change to `80% say that events and training are the main source...`

2. Less important: `the main source of info for accessibility of course content...` might read more professional as: `the main source of information regarding accessibility of course content...`